specs/corla.pvs

```diff
@@ -0,0 +1,923 @@
+%%% model of the Colorado Risk Limiting Audit (CORLA) tool.
```

Comment:
This sounds reasonable... But I'm going over corla.pvs now and already found an issue (the axioms county_names_unique and county_numbers_unique fail to specify that the two counties being compared must be in the same state). Assuming I want to fix that issue (which I do), I imagine I should do that in the domain-model-review branch rather than just talking about it here... or was that not what you guys envisioned for the review process? 

(I'll add a comment on that specific line as a review comment to this PR in a minute)